app/content/carto-user-manual/settings/understanding-your-organization-quotas.md

```diff
@@ -0,0 +1,77 @@
+## Understanding your organization quotas
+
+Every CARTO pricing plan includes a set of available quotas such as the total number of editors or the total number of applications that the organization can create.
+
+Let's review these quotas more in depth, with examples to help you predict usage.
+
+![Appearance Settings](/img/cloud-native-workspace/settings/settings_subscription_quotas.png)
+
+#### Editor users
+
+An editor is a user that has permissions to create and edit maps, connections, applications... This is a hard quota, and will be enforced whenever you try to add new editors. For more information on how to manage users, check [Managing User Roles](/carto-user-manual/settings/managing-user-roles/).
+
+{{% bannerNote title="Example"%}}
+If I already have 3 out of 3 editors available, I won't be able to invite new editors or promote viewers to editors.
+{{%/ bannerNote %}}
+
```

Comment:
We could also add admin, since they are also allowed to change their role to editor: "...or promove admin / viewers to editors"